app/views/users/index.html.erb

```diff
@@ -1,16 +1,129 @@
-<h1>All Users</h1>
-
-<ul class="users-cont">
-  <% @users.each do |user| %>
-    <li class="u-container">
-    <%= image_tag user.photo || "", class: "empty-square" %>
-    <div class="u-info">
-    <h3><%= link_to user.name, user %></h3>
-
-      <br>
-      Number of posts: <%= user.posts_counter %>
-    </div>
-      
-    </li>
-  <% end %>
-</ul>
+<style>
+  .container {
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 15px;
+  }
+
+  .text-center {
+    text-align: center;
+  }
+
+  .my-4 {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+
+  .row {
+    display: flex;
+    flex-wrap: wrap;
+    margin: -0.5rem;
+  }
+
+  .col-md-4 {
+    flex-basis: 33.33333%;
+    max-width: 33.33333%;
+    padding: 0.5rem;
+    box-sizing: border-box;
+  }
+
+  .card {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    background-color: #343a40;
+    color: #fff;
+    border-radius: 0.5rem;
+    overflow: hidden;
+  }
+
+  .card-img-top {
+    display: block;
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+  }
+
+  .card-body {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 1rem;
+  }
+
+  .card-title {
+    margin-bottom: 0.5rem;
+    font-size: 1.25rem;
+    font-weight: 500;
+    line-height: 1.2;
+  }
+
+  .card-text {
+    margin-bottom: 0;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  .card-footer {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    padding: 0.5rem;
+    background-color: #007bff;
+    border-radius: 0.5rem;
+  }
+
+  .btn {
+    display: inline-block;
+    font-weight: 400;
+    color: #fff;
+    text-align: center;
+    vertical-align: middle;
+    user-select: none;
+    background-color: transparent;
+    border: 1px solid transparent;
+    padding: 0.375rem 0.75rem;
+    font-size: 1rem;
+    line-height: 1.5;
+    border-radius: 0.5rem;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out,
+      border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  }
+
+  .btn:hover {
+    color: #007bff;
+    background-color: #fff;
+    border-color: #007bff;
+  }
+
+  @media (max-width: 767.98px) {
+    .col-md-4 {
+      flex-basis: 50%;
+      max-width: 50%;
+    }
+  }
+</style>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job adding all the changes recommended by the previous reviewer. However I think it would be better if you move all your styling code in a specific styling file inside your `app/assets/stylesheets/` directory. This for best practices reasons.